### PR TITLE
Document completion command

### PR DESCRIPTION
**Pull request by bucanero**
Fixes SUCCESS-273

### CODE REVIEW

File: content/howto/completion.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+First, you need to know which shell you are running. If you don't know, you can execute the following command:
+```sh
+echo $0
+```
+
+and the output should look like:
+
+```sh
+omar@workstation:~/Desktop/dgraph$ echo $0
```
Comment (OmarAyo) [inline content/howto/completion.md:30]: maybe make this more generic like:
```
session@workstation:~$ echo $0
```

File: content/howto/completion.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+...
+..
+.
+```
+
+Currently, the generated file has 2 lines at the beginning that need to be removed, or else the script won't run properly.
+You can comment them out with a `#`, or you can easily remove them with the following command:
+
+```sh
+sed -i.bak '1d;2d' ~/dgraph-completion.sh
```
Comment (OmarAyo) [inline content/howto/completion.md:64]: nice one !

File: content/howto/completion.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+Finally, reload the `bashrc` settings with the following command:
+
+```sh
+source ~/.bashrc
+```
+
+Now you can start typing `dgraph` and press <kbd>tab</kbd> to get auto-completion and suggestions:
+
+```txt
+omar@workstation:~/Desktop/dgraph$ dgraph
```
Comment (OmarAyo) [inline content/howto/completion.md:98]: also here make the shell name generic

File: content/howto/completion.md [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+
+First, you need to know which shell you are running. If you don't know, you can execute the following command:
+```sh
+echo $0
+```
+
+and the output should look like:
+
+```sh
+user@workstation:~/Desktop/dgraph$ echo $0
```
Comment (bucanero) [inline content/howto/completion.md:30]:
```suggestion
user@workstation:~/dgraph$ echo $0
```

File: content/howto/completion.md [hunk 5]
```diff
@@ -0,0 +1,128 @@
...
+Finally, reload the `bashrc` settings with the following command:
+
+```sh
+source ~/.bashrc
+```
+
+Now you can start typing `dgraph` and press <kbd>tab</kbd> to get auto-completion and suggestions:
+
+```txt
+user@workstation:~/Desktop/dgraph$ dgraph
```
Comment (bucanero) [inline content/howto/completion.md:98]:
```suggestion
user@workstation:~/dgraph$ dgraph 
```

File: content/howto/completion.md (hunk 1 above)
Comment (bucanero) [reply content/howto/completion.md]: fixed 👍

**Comment by bucanero** [comment]
As discussed, I've removed `zsh` instructions so we can move forward with this article.

File: content/howto/completion.md [hunk 6]
```diff
@@ -0,0 +1,102 @@
++++
+date = "2017-03-20T22:25:17+11:00"
+title = "Shell Completion"
```
Comment (aaroncarey) [inline content/howto/completion.md:3]: Between "date" and "title", please add the following for SEO purpose :)

description = "Dgraph supports command-line completion, a common feature provided by shells like `bash` or `zsh` that helps you to type commands in a fast and easy way.

File: content/howto/completion.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+weight = 1
+[menu.main]
+    parent = "howto"
++++
+
+Command-line completion is a common feature provided by shells like `bash` or `zsh` that helps users type commands in a fast and easy way.
```
Comment (aaroncarey) [inline content/howto/completion.md:9]: suggestion:
helps users type -> helps

**Review by aaroncarey — APPROVED** [review]
LGTM with two small tweaks requested :)

File: content/howto/completion.md (hunk 6 above)
Comment (bucanero) [reply content/howto/completion.md:4]: fixed

**Pull request merged**